tools/downloadDatasets.sh

```diff
@@ -22,9 +22,16 @@ downloadSS () {
 			wget ${2} || exit 1
 		fi
 		if [ ! -f ${1}/${1}.mtx ]; then
-			tar xf ${1}.tar.gz || exit 1
+			if [ -f ${1}.tar.gz ]; then
+				tar xf ${1}.tar.gz || exit 1
+			elif [ -f ${1}.gz ]; then
+				gzip -d ${1}.gz || exit 1
```

Comment:
@anyzelman this should be functional and already tested by tests/smoke/smoketests.sh. The downloaded file is  rndHermit256.mtx, not .gz ot .tar.gz file.

Reply:
I don't think the script downloads a mixture of .mtx, .tar.gz, and .gz files right? (What I mean is, was each case of the if-else-else checked?)

If not, I could just revert back to the `develop~HEAD` version of this script

Reply:
I just added one more level of nested if-else-else into an old if, where  tar and gzip are used for different files. This was not even used since .mtx file was downloaded.

Please revert this to `develop~HEAD`  version.